apps/studio/pages/project/[ref]/database/backups/restore-to-new-project.tsx

```diff
@@ -0,0 +1,510 @@
+    {
+      projectRef: project?.ref,
+    },
+    {
+      enabled: backupsSuccess && !isFreePlan,
+    }
+  )
+
+  const plan = subscription?.plan?.id
+  const isActiveHealthy = project?.status === PROJECT_STATUS.ACTIVE_HEALTHY
+
+  const isPermissionsLoaded = usePermissionsLoaded()
+  const canReadPhysicalBackups = useCheckPermissions(PermissionAction.READ, 'physical_backups')
+  const hasBackups = !!data?.backups.length
+  const canTriggerPhysicalBackups = useCheckPermissions(
+    PermissionAction.INFRA_EXECUTE,
+    'queue_job.restore.prepare'
+  )
+  const hasPITREnabled = backups?.pitr_enabled
+
+  const dbVersion = getDatabaseMajorVersion(project?.dbVersion ?? '')
+  const IS_PG15_OR_ABOVE = dbVersion >= 15
+  const PHYSICAL_BACKUPS_ENABLED = project?.is_physical_backups_enabled
+
+  const { data: cloneStatus } = useCloneStatusQuery({ projectRef: project?.ref })
+  const [selectedBackupId, setSelectedBackupId] = useState<number | null>(null)
+  const [showConfirmationDialog, setShowConfirmationDialog] = useState(false)
+  const [showNewProjectDialog, setShowNewProjectDialog] = useState(false)
+
+  const isLoading = isPermissionsLoaded && cloneBackupsLoading && backupsLoading
+
+  const { mutate: triggerClone } = useProjectCloneMutation({
+    onError: (error) => {
+      console.error('error', error)
+      toast.error('Failed to restore to new project')
+    },
+    onSuccess: () => {
+      toast.success('Restoration process started')
+    },
+  })
+
+  const FormSchema = z.object({
+    name: z.string().min(1),
+    password: z.string().min(1),
+  })
+
+  const form = useForm<z.infer<typeof FormSchema>>({
+    resolver: zodResolver(FormSchema),
+    defaultValues: {
+      name: '',
+      password: '',
+    },
+  })
+
+  const [passwordStrengthScore, setPasswordStrengthScore] = useState(0)
+  const [passwordStrengthMessage, setPasswordStrengthMessage] = useState('')
+
+  const delayedCheckPasswordStrength = useRef(
+    debounce((value: string) => checkPasswordStrength(value), 300)
+  ).current
+
+  async function checkPasswordStrength(value: string) {
+    const { message, strength } = await passwordStrength(value)
+    setPasswordStrengthScore(strength)
+    setPasswordStrengthMessage(message)
+  }
+
+  function generateStrongPassword() {
+    const password = generator.generate({
+      length: 16,
+      numbers: true,
+      uppercase: true,
+    })
+
+    form.setValue('password', password)
+    delayedCheckPasswordStrength(password)
+  }
+
+  if (isLoading) {
+    return <GenericSkeletonLoader />
+  }
+
+  if (!canReadPhysicalBackups) {
+    return <NoPermission resourceText="view backups" />
+  }
+
+  if (!canTriggerPhysicalBackups) {
+    return <NoPermission resourceText="restore backups" />
+  }
+
+  if (!IS_PG15_OR_ABOVE) {
+    return (
+      <Admonition
+        type="default"
+        title="Restore to new project is not available for this database version"
+        description={
+          <>
```

Comment:
let's use Markdown (from `components/interfaces/Markdown`) for this - so the link automatically gets styled nicely